Sources_v3/Database/DTOs/CurrentUserDTO.swift

```diff
@@ -83,6 +84,42 @@ extension NSManagedObjectContext: CurrentUserDatabaseSession {
         dto.unreadMessagesCount = Int16(count.messages)
     }
     
+    func saveCurrentUserDevice(_ device: DevicePayload) throws {
+        guard let currentUser = currentUser() else {
+            throw ClientError.CurrentUserDoesNotExist()
+        }
+        
+        saveDevice(device, for: currentUser)
```

Comment:
nit: maybe we can we reuse `try saveCurrentUserDevices([device])` here?

Reply:
Reworked in 107624943165ad39afd991a7fd9aba8d58c1a28a